Revealed/Features/PostDetail/PostDetailView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  PostDetailView.swift
+//  Revealed
+//
+//  Created by KokHong on 24/3/20.
+//  Copyright © 2020 Pointwelve. All rights reserved.
+//
+
+import SwiftUI
+
+struct PostDetailView: View {
+  let post: PostDetail
+  let padding = CGFloat.padding
+  let leadTrailingEdges: Edge.Set = [.leading, .trailing]
+
+  init(post: PostDetail) {
+    self.post = post
+  }
+
+  var body: some View {
+    VStack(alignment: HorizontalAlignment.leading, spacing: padding) {
+      // Header Text
+      //TODO: Change NTUC to organisation
+      Text("""
+      NTUC・\(post.author.username)\
+      ・\(Date(timeIntervalSince1970: Double(post.createdAt)).relativeDateString)
+      """).font(.caption).padding(leadTrailingEdges, padding)
+
+      // Subject
+      Text(post.subject).font(.title).lineLimit(2).padding(leadTrailingEdges, padding)
+
+      // Content
+      Text(post.content).font(.body).foregroundColor(.charcoalGrey60).padding(leadTrailingEdges, padding)
```

Comment:
foreground color should be `.secondary`

Reply:
we shouldnt have to set the padding for each component here, u can either group it and set the padding, or just set it on the VStack

Reply:
Ya initially i set it to VStack directly, because the CommentHeaderView need full width, so i separate into each component. Can group it together using another VStack?

Reply:
> foreground color should be `.secondary`

okay. what about that CommentHeaderView Background color?

Reply:
> Ya initially i set it to VStack directly, because the CommentHeaderView need full width, so i separate into each component. Can group it together using another VStack?

Ya you can, or use `Group {}` i think works too